WorkflowConcurrency/Sources/AsyncOperationWorker.swift

```diff
@@ -16,29 +16,23 @@
 
 import Foundation
 import Workflow
-import WorkflowTesting
-import XCTest
-@testable import WorkflowConcurrencyTesting
 
+/// A worker for executing an async operation
```

Comment:
I've added example usage.  I can't think of a way to re-run the async closure offhand to document since the consumer doesn't have the ability to change `isEquivalent` for the worker.